### PR TITLE
808 Add SPM support

### PR DESCRIPTION
**Pull request by Antonijo2307**
Implemented package.swift for "case" target. Case covers both plain and case...
min version set to ios11
808 Add SPM support

### CODE REVIEW

**Review by vlaho-pd — APPROVED** [review]

**Review by Truba — APPROVED** [review]

File: SwiftI18n/Classes/Main/Localizable.swift [hunk 1]
```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import UIKit
```
Comment (nikolamajcen) [inline SwiftI18n/Classes/Main/Localizable.swift:10]: If you added `UIKit`, you can remove `Foundation`. But that's not a big deal.

**Review by nikolamajcen — APPROVED** [review]

File: SwiftI18n/Classes/Views/SwiftUI/LocText.swift [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    public var text: Text {
+        Text(I18nManager.instance.localizationPerformingBlock(key, language))
+    }
+    
+    public var body: some View { text }
+}
+
+public extension LocText {
+    
+    init(_ key: String) {
```
Comment (jabou) [inline SwiftI18n/Classes/Views/SwiftUI/LocText.swift:54]: Please remind me. Did we say that we will set this to `init(localizationKey: String)` from here on will we leave this to the lib implementor?

**Review by jabou — APPROVED** [review]

**Comment by jabou** [comment]
@Antonijo2307 can you please check open comments? Let's finalize and merge this PR as more and more team members will need SwiftUI support that is included within this PR.

File: SwiftI18n/Classes/Views/SwiftUI/LocText.swift (hunk 2 above)
Comment (Antonijo2307) [reply SwiftI18n/Classes/Views/SwiftUI/LocText.swift:54]: We left it to the lib implementor as I recall

**Pull request merged**